src/main/java/org/team1540/robot2024/Robot.java

```diff
@@ -122,6 +124,7 @@ public void robotPeriodic() {
         // the Command-based framework to work.
         CommandScheduler.getInstance().run();
         if (Constants.currentMode == Constants.Mode.REAL) robotContainer.odometrySignalRefresher.periodic();
+        AlertManager.getInstance().update();
```

Comment:
Does this take any considerable amount of time with the NT stuff? (wpilib apparently has profiling tools somewhere)

Reply:
The alerts don't publish very much stuff to NT, so this shouldn't take a significant amount of time. We could profile it if we see loop overruns because of `Smartdashboard.updateValues()`